src/components/Calculator.jsx

```diff
@@ -2,6 +2,7 @@ import { useState } from 'react';
 import Result from './Result';
 import Keys from './keys';
 import calculate from '../logic/calculate';
+import Quote from './Quote';
```

Comment:
- [OPTIOANL] Great job so far 👏🏻 However, The `<Quotes>` component should be imported into the main component which is the App component, and not the Calculator. please change this. 💯 👍🏻 
- Your code should look like this:

```jsx
import Calculator from "./components/Calculator"
import Quotes from "./components/Quotes"

function App() {
  return (
    <>
      <Calculator />
      <Quotes />
    </>
  )
}

export default App
```